main.py

```diff
@@ -84,7 +84,10 @@ def remove_stop_words(frequency_list):
 	search_mode = False
 
 url = wikipedia_api_link + string_query
-response = requests.get(url)
+try:
+    response = requests.get(url)
+except Exception, e:
```

Comment:
Even if there was an exception, the rest of code is getting executed, raising different error

Maybe you can shift the whole code in try statement or find some other work around